docs/source/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'osier'
+copyright = '2022, Samuel Dotson'
+author = 'Samuel Dotson'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "myst_parser",
+    "numpydoc",
+    "sphinxcontrib.napoleon",
```

Comment:
```suggestion
    "sphinx.ext.napoleon",
```
When running `make html` after installing `osier` with Python 3.10, there is an error:

```python
Extension error:
Could not import extension sphinxcontrib.napoleon (exception: cannot import name 'Callable' from 'collections' (/home/ooblack/miniconda3/envs/osier-env/lib/python3.10/collections/__init__.py))
```

It turns out that [Python 3.10 moved the `Callable` object from the `collections` module to the `typing` module](https://docs.python.org/3/whatsnew/3.10.html#pep-612-parameter-specification-variables). Looking at the [PyPI page for `sphinxcontrib-napoleon`](https://pypi.org/project/sphinxcontrib-napoleon/), we can see that this package is now included with `sphinx` after version 1.3 as `sphinx.ext.napoleon`. Making this change, the build runs successfully.

Reply:
Just a heads up, there are two warning messages I get when running `make html` after making this change:

```
WARNING: html_static_path entry '_static' does not exist
```
and
```
/home/ooblack/projects/osier/README.md:44: WARNING: 'myst' reference target not found: docs/source/contrib.md
```

[Here](https://gist.github.com/yardasol/fcb04b0187902fb780412c765da807ba) is the full output

Reply:
Thanks for the `napoleon` update! Regarding the two warnings you receive, they're both ignorable for now. 
* The _static folder doesn't exist because I don't have any static files and git won't let you push an empty directory. 
* The `myst` reference warning is a bit strange and may require a separate issue. The explanation is that I prefer to have a single contrib file rather than a contrib file in the docs that references another contributing file. I'm not sure what the best way to handle this is, the only difference is that the README has a hyperlink to the contrib file when viewed on github, while the docs has a hyperlink immediately below the reference.